example/hashes/hashesserver.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"crypto/sha1"
+	"fmt"
+	"hash"
+	"io"
+	"net"
+	"os"
+
+	"capnproto.org/go/capnp/v3/rpc"
+	"capnproto.org/go/capnp/v3/server"
+	"golang.org/x/net/context"
```

Comment:
Shouldn't this just be the `"context"`?

Reply:
fixed in 84a7de0a116bd09eb7487daf16cea767f7d94403